user/how-to-guides/how-to-organize-your-qubes.md

```diff
@@ -0,0 +1,451 @@
+  just convenient) to make edits to config files in the root filesystem, and
+  she'd rather not have to worry about losing those changes during an app qube
+  reboot. She knows that she could use [bind-dirs](/doc/bind-dirs/) to make
+  those changes persistent, but sometimes she doesn't want to get bogged down
+  doing with all that and figures it wouldn't be worth it just for this one
+  qube. She's secretly glad that Qubes OS doesn't judge her this and just gives
+  her the freedom to do things however she likes while keeping everything
+  securely compartmentalized. At times like these, she takes comfort in knowing
+  that things can be messy and disorganized *within* a qube while her overall
+  digital life remains well-organized.
+
+- Several email qubes. Since Alice is a serious programmer, she likes to use a
+  command-line mail client, so both her work and personal email qubes are based
+  on a template with
+  [Mutt](https://github.com/Qubes-Community/Contents/blob/master/docs/configuration/mutt.md)
+  installed. The email qubes where she sends and receives PGP-signed and
+  encrypted email access her PGP backend qube (more on that below). For
+  security, she configured Mutt to open all attachments she receives in
+  [disposable qubes](/doc/how-to-use-disposables/).
+
+- Several qubes for communication tools, like Signal, Slack, Zoom, Telegram,
+  IRC, and Discord. This is where she teleconferences and chats with clients.
+  She uses [USB passthrough](/doc/how-to-use-usb-devices/) to attach her webcam
+  to each qube as needed and detaches it afterward. Likewise, she gives each
+  qube access to her microphone while it's needed, then removes access
+  afterward. This way, she doesn't have to trust any given video chat program's
+  mute button and doesn't have to worry about being spied on when she's not on
+  a call. She also has a qube for social media platforms like Twitter, Reddit,
+  and Hacker News for networking and keeping up with new developments (or so
+  she claims; in reality, it's mostly for feuds over programming language
+  superiority, Vim vs. Emacs wars, and tabs vs. spaces crusades).
+
+- A backend PGP vault. This offline qube holds her PGP code signing keys and is
+  securely shared among several projects. Only the frontend qubes she
+  explicitly authorizes have access to this qube, and even then, they only have
+  access through the secure [Split GPG](/doc/split-gpg/) system so that her
+  private keys aren't at risk.
+
+- A password manager vault. This is where she runs her offline password manager
+  for logging into everything.
+
+- Personal qubes. One of the things Alice loves the most about Qubes is that
+  she can use it for both work *and* personal stuff without having to worry
+  about cross-contamination. Accordingly, she has several qubes that pertain to
+  her personal life. For example, she has a vault that holds her medical
+  documents, test results, and vaccination records. She has another vault for
```

Comment:
I'd mention here that the vaults are non-networked? I think it is implicit, but I think it could be nice to make it explicit.

Reply:
Good point! Clarified this throughout the document.